external/untwine/bu/BuPyramid.cpp

```diff
@@ -13,6 +14,36 @@
 #include "../untwine/Common.hpp"
 #include "../untwine/ProgressWriter.hpp"
 
+namespace
+{
+
+// PDAL's directoryList had a bug, so we've imported a working
+// version here so that we can still use older PDAL releases.
+std::vector<std::string> directoryList(const std::string& dir)
+{
+    namespace fs = std::filesystem;
+
+    std::vector<std::string> files;
+
+    try
+    {
+        fs::directory_iterator it(untwine::toNative(dir));
```

Comment:
this breaks build on macos when the target is less than 10.15 which is not realistic for now.
Any chance to bring a code which would compile on 10.13?
see https://github.com/qgis/QGIS-Mac-Packager/pull/150

Reply:
@hobu @abellgithub Unfortunately it looks like the recent untwine update breaks QGIS macOS build. Would it be possible to update untwine code so that it compiles on macos 10.13 as well?

Reply:
Hopefully this fix from @hobu should help https://github.com/qgis/QGIS/pull/48697